packages/google_identity_services_web/example/integration_test/js_interop_id_test.dart

```diff
@@ -37,45 +40,46 @@ void main() async {
       final IdConfiguration config = IdConfiguration(
         client_id: 'testing_1-2-3',
         auto_select: false,
-        callback: allowInterop((_) {}),
+        callback: (_) {},
```

Comment:
Wouldn't this and below come across the issue of the parameter being `dynamic` and therefore not supported by `toJS`? Maybe this is handled elsewhere and if so, ignore me. :)

Reply:
@srujzs, having several of this working but not the other did confuse me. I convinced myself that this works because the parameter is a JSObject that I've defined in the JS-interop layer (but so is "Event"?)

If you can't explain me why this works but the other doesn't, there's a problem! :P

Reply:
I just replaced `(_) {...` with `(JSAny? _) {...` and it worked